src/main/java/use_cases/set_availability/SetAvailabilityModel.java

```diff
@@ -0,0 +1,20 @@
+package use_cases.set_availability;
+
+/**
+ * A model class for packaging the required info needed by the setAvailabilityPresenter.
+ *
+ * @author od-obas1187
+ */
+public class SetAvailabilityModel {
+
+    private final String curriculumName;
```

Comment:
I'm not fully sure as to the method of display this use case will have, but simply a curriculum name seems lacking. Get in contact with Amir as soon as possible to begin hammering out what this model should actually include, based on what Amir intends to do with your use case, UI-side

Reply:
Yes, it would be ideal for Odosa and I to meet up and discuss how I can tailor the UI to fit his already modified code